.idea/compiler.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you confirm that you can check out the project from scratch and load in intelliJ just fine? we have to protect the process for loading the training app onto a new computer as it is very expensive in many classes.